### DISCUSSION TITLE
"CTkInputDialog" default input/placeholder text

### DISCUSSION THREAD

**Discussion by Guillaume-Gibault** [TomSchimansky/CustomTkinter#1352] [Q&A]
Hello, I'm new to this library and I'd to know if there was a way to be able to change the default input/placeholder text of the "CTkInputDialog" widget or if someone had a custom widget of this kind. 
Thanks in advance!

**Comment by AlbertSolomon** [comment]
@Guillaume-Gibault 👋🏾 In order to change the input/placeholder text, you have to edit the text attribute like so
```diff python
   
- dialog = ctk.CTkInputDialog(text="new thing:", title="Test")
+ dialog = ctk.CTkInputDialog(text="something else", title="Test")

# The same applies to the title attribute
```
Please don't forget to mark this as answered if it has been helpful 😅

**Reply by Guillaume-Gibault** [reply]
Thanks for the reply and sorry for the late response.
This isn't exactly what I was trying to do, what I looked for was a grayed "example text" that would be behind the input zone.
